packages/zilliqa-js-account/package.json

```diff
@@ -38,6 +38,6 @@
     "web3": "^1.0.0-beta.36"
   },
   "engines": {
-    "node": ">=10.0.0 <=12.12.0"
+    "node": ">=10.0.0 <=12.16.3"
```

Comment:
Is it possible to specify `< 13.0.0` insted so you don't need to regularly update to latest?

cc @bb111189

Reply:
hey. I just saw this. Let me discuss with my team to see whether they are ok with it or not